src/pyedb/dotnet/edb_core/cell/layout.py

```diff
@@ -323,7 +323,11 @@ def find_net_by_name(self, value: str):
 
         """
         obj = self._pedb._edb.Cell.Net.FindByName(self._edb_object, value)
```

Comment:
Cannot the test `is_null` be performed sooner on `obj` and raise the `ValueError` if needed?